Jellyfin.Api/Controllers/MediaInfoController.cs

```diff
@@ -111,18 +114,17 @@ public async Task<ActionResult<PlaybackInfoResponse>> GetPlaybackInfo([FromRoute
             [FromQuery] int? maxAudioChannels,
             [FromQuery] string? mediaSourceId,
             [FromQuery] string? liveStreamId,
-            [FromBody] DeviceProfileDto? deviceProfile,
-            [FromQuery] bool autoOpenLiveStream = false,
-            [FromQuery] bool enableDirectPlay = true,
-            [FromQuery] bool enableDirectStream = true,
-            [FromQuery] bool enableTranscoding = true,
-            [FromQuery] bool allowVideoStreamCopy = true,
-            [FromQuery] bool allowAudioStreamCopy = true)
+            [FromQuery] bool? autoOpenLiveStream,
+            [FromQuery] bool? enableDirectPlay,
+            [FromQuery] bool? enableDirectStream,
+            [FromQuery] bool? enableTranscoding,
+            [FromQuery] bool? allowVideoStreamCopy,
+            [FromQuery] bool? allowAudioStreamCopy,
+            [FromBody] PlaybackInfoDto? playbackInfoDto)
         {
             var authInfo = _authContext.GetAuthorizationInfo(Request);
 
-            var profile = deviceProfile?.DeviceProfile;
-
+            var profile = playbackInfoDto?.DeviceProfile?.DeviceProfile;
```

Comment:
That doesn't look right…
Does this move the device profile one level deeper in the JSON payload?